README.md

```diff
@@ -1273,9 +1273,9 @@ if err != nil {
 You can impersonate to another user
 The impersonator user must have the `impersonation` permission in order for this request to work.
 The response would be a refresh JWT of the impersonated user
-
+TenantID would be the tenant to set as DCT claim, in case set 
 ```go
-refreshJWT, err := descopeClient.Management.JWT().Impersonate(context.Background(), "impersonator id", "login id", true)
+refreshJWT, err := descopeClient.Management.JWT().Impersonate(context.Background(), "impersonator id", "login id", true, map[string]any{"k1":"v1", "T1"})
```

Comment:
I think you have syntax issue in the map, also, please add the custom claims to the comment

Reply:
Done